bots/task/test-task

```diff
@@ -27,6 +27,7 @@ import os
 import signal
 import shutil
 import tempfile
+import mock
```

Comment:
that's the external py2 variant, use `from unittest.mock import patch` to use the python standard lib one.